.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+
+## Fixes 
+<!-- For e.g Fixes: https://issues.redhat.com/browse/HAC-XXX -->
+
+
+## Description
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+
+## Type of change
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Feature
+- [ ] Bugfix
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] Documentation content changes
+- [ ] Other (please describe):
+
+## Screen shots / Gifs for design review 
+<!-- If change affects UI in any way, tag relevant UX people and add screenshots/gifs  -->
+
+
+## How Has This Been Tested?
+<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration -->
+
+- [ ] Test A
```

Comment:
Not sure if a checklist will be helpful here.  Maybe just a comment to reinforce the steps needed to reproduce the issue/feature.

Reply:
Added in a comment.